scripts/CheckNetworkMultiSigConfigs.s.sol

```diff
@@ -0,0 +1,179 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import {console2} from "forge-std/console2.sol";
+import {Script} from "forge-std/Script.sol";
+import {VmSafe} from "forge-std/Vm.sol";
+
+/**
+ * @title CheckNetworkMultiSigConfigs
+ * @notice A script to check security configurations of important multisigs for a given network.
+ *         The usage is as follows:
+ *         $ forge script CheckNetworkMultiSigConfigs --rpc-url $MAINNET_RPC_URL
+ */
+contract CheckNetworkMultiSigConfigs is Script {
+    struct MultiSigConfig {
+        address multiSigAddress;
+        uint256 threshold;
+        uint256 totalSigners;
+        address[] signers;
+    }
+
+    MultiSigConfig[] multiSigConfigs;
+
+    bool internal hasErrors;
+
+    /**
+     * @notice The entrypoint function.
+     */
+    function run() public {
+        string memory network;
+        if (block.chainid == 1) {
+            network = "mainnet";
+        } else if (block.chainid == 11155111) {
+            network = "sepolia";
+        } else if (block.chainid == 5) {
+            network = "goerli";
+        } else {
+            revert(
+                string.concat(
+                    "Unsupported chain ID: ",
+                    vm.toString(block.chainid),
+                    ". Please call runOnDir(string,bool) directly."
+                )
+            );
+        }
+        string memory jsonDir = string.concat("superchain/extra/addresses/", network);
+        runOnDir(jsonDir);
+    }
```

Comment:
Consider using forge-std's [StdChains](https://github.com/foundry-rs/forge-std/blob/master/src/StdChains.sol) to map from chain ID to network name using `getChain(chainId)`